test/unittests/evm_eof_test.cpp

```diff
@@ -203,17 +208,368 @@ TEST_P(evm, eof1_codecopy_out_of_bounds)
+}
+
+TEST_P(evm, eof1_rjumpv_long_jumps)
+{
+    // Relative jumps are not implemented in Advanced.
+    if (is_advanced())
+        return;
+
+    rev = EVMC_CANCUN;
+    auto code =
+        rjump(0x7fff - 3 - 5) + (0x7fff - 3 - 2 - 8 - 5) * bytecode{OP_JUMPDEST} + 7 + ret_top();
+
+    code += rjumpv({-0x7FFF, 0x7FFF - 8 - 2 - 8}, 0) +
+            (0x7fff - 8 - 2 - 8) * bytecode{OP_JUMPDEST} + 5 + ret_top();
+
+    code = eof1_bytecode(code, 2);
+    auto& rjumpv_cond = code[0x7fff - 3 - 5 + 3 + 1 + 19];
+
+    execute(code);
+    EXPECT_STATUS(EVMC_SUCCESS);
+    EXPECT_OUTPUT_INT(7);
+
+    rjumpv_cond = 1;
+
+    execute(code);
+    EXPECT_STATUS(EVMC_SUCCESS);
+    EXPECT_OUTPUT_INT(5);
+}
+
+TEST_P(evm, relative_jumps_undefined_in_legacy)
+{
+    rev = EVMC_CANCUN;
+    auto code = rjump(1) + OP_INVALID + mstore8(0, 1) + ret(0, 1);
+
+    execute(code);
+    EXPECT_STATUS(EVMC_UNDEFINED_INSTRUCTION);
+
+    code = rjumpi(10, 1) + mstore8(0, 2) + ret(0, 1) + mstore8(0, 1) + ret(0, 1);
+
+    execute(code);
+    EXPECT_STATUS(EVMC_UNDEFINED_INSTRUCTION);
+}
+
+TEST_P(evm, eof_function_example1)
+{
+    // Relative jumps are not implemented in Advanced.
+    if (is_advanced())
+        return;
+
+    rev = EVMC_CANCUN;
+    const auto code =
+        "EF00 01 010008 020002 000f 0002 030000 00"
+        "00000002 02010002"
+        "6001 6008 b00001 " +
+        ret_top() + "03b1";
+
+    ASSERT_EQ((int)evmone::validate_eof(rev, code), (int)evmone::EOFValidationError{});
+
+    execute(code);
+    EXPECT_GAS_USED(EVMC_SUCCESS, 32);
+    EXPECT_OUTPUT_INT(7);
+}
+
+TEST_P(evm, eof_function_example2)
+{
+    // Relative jumps are not implemented in Advanced.
+    if (is_advanced())
+        return;
+
+    rev = EVMC_CANCUN;
+    const auto code =
+        "ef0001 01000c 020003 003b 0017 001d 030000 00 00000004 01010003 01010004"
+        "60043560003560e01c63c766526781145d001c63c6c2ea1781145d00065050600080fd50b00002600052602060"
+        "00f350b0000160005260206000f3"
+        "600181115d0004506001b160018103b0000181029050b1"
+        "600281115d0004506001b160028103b0000260018203b00002019050b1"_hex;
+
+    ASSERT_EQ((int)evmone::validate_eof(rev, code), (int)evmone::EOFValidationError{});
+
+    // Call fac(5)
+    const auto calldata_fac =
+        "c76652670000000000000000000000000000000000000000000000000000000000000005"_hex;
+    execute(bytecode{code}, calldata_fac);
+    EXPECT_GAS_USED(EVMC_SUCCESS, 246);
+    EXPECT_EQ(output, "0000000000000000000000000000000000000000000000000000000000000078"_hex);
+
+    // Call fib(15)
+    const auto calldata_fib =
+        "c6c2ea17000000000000000000000000000000000000000000000000000000000000000f"_hex;
+    execute(bytecode{code}, calldata_fib);
+    EXPECT_GAS_USED(EVMC_SUCCESS, 44544);
+    EXPECT_EQ(output, "0000000000000000000000000000000000000000000000000000000000000262"_hex);
+}
+
+TEST_P(evm, eof_data_only_contract)
+{
+    if (is_advanced())
```

Comment:
Comment missing here.

Reply:
All these tests works in `advanced` I removed the `if`